README.md

```diff
@@ -1,6 +1,7 @@
 # sns-quill
 
-Cold wallet toolkit for interacting with a Service Nervous System's Ledger & Governance canisters.
+Cold wallet toolkit for interacting with the ledger and governance canister of a service nervous system (SNS).
```

Comment:
Are we not capitalizing service nervous system anymore?

Reply:
no.
I thought we never did (happy to be proven wrong as you are not the first person to be of a different opinion). 
I think we have many servous nervous systems but "the" Network Nervous System, thus the former is just a concept, the latter is "the one" thus capital. At least I thought these were the guidelines that we received at some point.